ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -269,7 +269,7 @@ presubmits:
         name: job-definition
     trigger: '(?m)^/test (?:.*? )?e2e-metal(?: .*?)?$'
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
the aws job doesn't run always but https://github.com/openshift/release/pull/3477/files#diff-faaaaebefcc08c8105ee06d2eaefba12R15

ie !docs

Reply:
fixed